htdocs/beta/controller/binet/request.php

```diff
@@ -1,14 +1,28 @@
 <?php
 
-  function not_sent() {
-    $request = select_request($_GET["request"], array("sent"));
-    header_if($request["sent"] == 1, 403);
+  function is_sendable($request) {
+    $request = select_request($request, array("sending_date", "wave"));
+    $wave = select_wave($request["wave"], array("published"));
+    return is_empty($request["sending_date"]) && !$wave["published"];
+  }
```

Comment:
J'avais une requête envoyé mais jamais étudiée, je l'ai étudiée une fois la vague expirée et je me suis attribué des subventions.
Le bon coté c'est que ces subventions ne sont pas utilisées du coup. Mais c'est bizarre quand même, ma requête et notée "accordée" dans les "Subventions"